linera-service/src/server.rs

```diff
@@ -558,6 +590,36 @@ async fn run(options: ServerOptions) {
                 .await
                 .unwrap();
         }
+
+        ServerCommand::EditShards {
+            server_config_path,
+            num_shards,
+            host,
+            port,
+            metrics_host,
+            metrics_port,
+        } => {
+            assert!(
+                num_shards > 0 && num_shards < 100,
+                "The number of shards should be between 1 and 99"
+            );
+            let mut server_config: ValidatorServerConfig =
+                util::read_json(&server_config_path).expect("Fail to read server config");
+            let mut shards = Vec::new();
+            for i in 0..num_shards {
+                let index = format!("{i:2}");
+                let shard = ShardConfig {
+                    host: host.replace("%%", &index),
```

Comment:
Unfortunately, Docker compose begins indexing service replicas from 1. So the first shard is `docker-shard-1`.

Reply:
Good catch. I meant to start at `1`